inst/rmarkdown/templates/transmissibility-report/skeleton/rmdchunks/EpiNow2.Rmd

```diff
@@ -1,7 +1,7 @@
 ## _EpiNow2_ 
 
 ```{r}
-library("EpiNow2")
+pacman::p_load("EpiNow2")
```

Comment:
I guess the explicit namespacing from `pacman` is to avoid loading it for a single use in this and other vignettes. Is it useful to consider the R literacy of the readers and explain explicit namespacing somewhere? You do say basic R literacy is required, but there might be broad variation within that.

Reply:
It is actually already loading as a side effect of `require("pacman")`. 

``` r
if (require("fortunes")) {
  
  fortune()
  
}
#> Loading required package: fortunes
#> 
#> Soham: How to compute the p-value of a statistic generally?
#> Berton Gunter: runif(1)
#>    -- Soham and Berton Gunter
#>       R-help (May 2010)
```

<sup>Created on 2022-10-27 with [reprex v2.0.2.9000](https://reprex.tidyverse.org)</sup>

I added the namespace because I find it useful to know where uncommon functions, that users might not know, come from but I didn't consider it might be confusing.

I don't think it'll be appropriate to discuss explicit namespacing it so if we decide it's not clear enough, I'd rather remove it. What do you think?